src/v1/index.js

```diff
@@ -38,6 +38,9 @@ const auth ={
       return {scheme: "basic", principal: username, credentials: password};
     }
   },
+    kerberos:(base64EncodedTicket)=> {
+    return {scheme:"kerberos", credentials: base64EncodedTicket};
```

Comment:
Please add tests for this method to avoid breaking with a typo or smth like it. We should also type check given `base64EncodedTicket` with `util.assertString()` function.